### ISSUE TITLE
Serve fonts from CDN for improved performance (low priority)

### ISSUE THREAD

**Issue by joshmorel** [FightPandemics/FightPandemics#1079]
From @robinv85 (#361 ):

> For the fonts, I think we don't even have the fonts correctly imported at the moment. There are 3 custom fonts, one didn't seem to work via npm install, and I question if we really need 3 custom fonts, one of them is currently only used on some buttons. This might have a big impact on performance. To import the fonts via CDN would be nice, we could just import them in index.html instead of currently adding them through npm install.

**Comment by robinv85** [comment]
yeah not sure if the info that I posted before is still accurate at the moment

**Comment by joshmorel** [comment]
Ah okay. @robinv85  feel free to edit the main comment or close this if not relevant.

Josh

**Comment by phuclb** [comment]
Beside that, we re-set font-family even it use the same global font.

For example (component FaqContainer, FootNote, CookieAlert,...):
```
body {
  font-family: "Poppins", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
}
```

and element:
```
.elementClass {
  font-family: Poppins;
}
```

1. Why do we have to declare a very long web font-stack live above?
2. Why don't we switch to Google Embed Font for better performance?

Thanks.

**Comment by robinv85** [comment]
@phuclb generally a lot of people worked on this code base and lots of code got merged without proper review or alignment with other developers... 
I also noticed that really long font stack before, can't remember who added that, it's not necessarily a big problem, but usually it would be enough to have only a few in there (although nowadays there are a lot of different browsers, operating systems etc.) so if you think you can improve that font stack and have a good way to check these fallbacks and compatibility across different environments, then you're welcome to contribute @phuclb 
If with "Google Embed Font" you mean using the Google Font API aka using the Google CDN to load the fonts, yeah this is exactly what is stated in this issue here...

**Comment by phuclb** [comment]
@robinv85, we can embed Google Font in via <link> tag, it's also a CDN load, no need to use API with script.

I can review and remove all redundant code (font-family), and shorten font stack if necessary, to improve performance, a bit.

Let me know final decision.
Thanks.
